frontend/uddhar-frontend/src/public/components/LogoScroll.jsx

```
@@ -0,0 +1,58 @@
+import { motion } from "framer-motion";
+
+const VerticalLogoScroll = ({ orgLogo }) => {
+  const scrollAnimation = {
+    animate: {
+      x: ["0%", "-50%"],
+      transition: {
+        duration: 20,
+        ease: "linear",
+        repeat: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)